ExternalBeamPlanning/Logic/vtkSlicerExternalBeamPlanningModuleLogic.cxx

```diff
@@ -200,7 +200,11 @@ void vtkSlicerExternalBeamPlanningModuleLogic::OnMRMLSceneEndImport()
   // Observe plan events of all plan nodes
   std::vector<vtkMRMLNode*> nodes;
   this->GetMRMLScene()->GetNodesByClass("vtkMRMLRTPlanNode", nodes);
+<<<<<<< HEAD
   for ( auto nodeIt=nodes.begin(); nodeIt!=nodes.end(); ++nodeIt)
+=======
+  for (auto nodeIt=nodes.begin(); nodeIt!=nodes.end(); ++nodeIt)
+>>>>>>> 54484851... ENH: MLC leaves projection recalculation and GUI elements upgrade

```

Comment:
This is a merge conflict. The code would fail to build like this.

Reply:
I've fixed that.